apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+import type { RouterOutputs } from "@agreeto/api";
+import { RadixColor } from "@agreeto/api/types";
+import { Button } from "@agreeto/ui";
+import * as AlertDialog from "@radix-ui/react-alert-dialog";
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import * as Tooltip from "@radix-ui/react-tooltip";
+import clsx from "clsx";
+import React, { type FC } from "react";
+import { AiOutlineMore } from "react-icons/ai";
+import { HiCheckCircle, HiOutlineExclamation, HiTrash } from "react-icons/hi";
+import resolveConfig from "tailwindcss/resolveConfig";
+
+import { trpcApi } from "~features/trpc/api/hooks";
+
+import tailwindConfig from "./../../../tailwind.config";
+
+type Account = RouterOutputs["user"]["myAccounts"]["accounts"][number];
+
+const AccountCard: FC<{
+  account: Account;
+}> = ({ account }) => {
+  const initials = account.email?.substring(0, 2).toLocaleUpperCase();
+
+  //   const { data: colors } = useGetAccountColors({ staleTime: 60 * 60 * 1000 });
+  // style the events
+  const fullConfig = resolveConfig({
+    ...tailwindConfig,
+    content: ["./src/**/*.{html,js,ts,tsx}"],
+  });
+  console.log(fullConfig);
+
+  const colors = Object.keys(RadixColor) as RadixColor[];
+  const [container, setContainer] = React.useState<HTMLDivElement | null>(null);
+
+  const themeColors = fullConfig.theme?.colors as Record<RadixColor, string>;
+  const utils = trpcApi.useContext();
+  const { mutateAsync: updateEventColor } =
+    trpcApi.account.updateColor.useMutation({
+      onSuccess() {
+        utils.user.myAccounts.invalidate();
+      },
+    });
+
+  return (
+    <Tooltip.Provider>
+      <div className="flex justify-between py-6 pr-6 space-x-4 border rounded-lg h-30 border-mauve-6 pl-9">
+        {/* Info */}
+        <div className="flex w-full gap-4">
+          {/* Avatar */}
+          <div
+            className="self-center w-10 h-10 leading-10 text-center rounded-full"
+            style={{
+              backgroundColor: themeColors?.[account.eventColor][7],
+              color: themeColors?.[account.eventColor][11],
+            }}
+          >
+            {initials}
+          </div>
+          {/* the account color for the given account */}
+          <div className="flex flex-col justify-center">
+            <div className="text-sm font-medium">{account.eventColor}</div>
+          </div>
+          {/* Email, Colors & Badges */}
+          {/* <div className="flex flex-grow"> */}
+          <div className="flex flex-col items-start flex-grow px-10 ">
+            {/* this div makes sure that the children have the same width as determined by colors */}
+            <div>
+              <div className="flex justify-between">
+                {/* a div displaying the email on the left side and a conditional organizer badge on the right */}
+                <div className="text-sm font-normal">{account.email}</div>
```

Comment:
maybe an icon of the provider before the email or something that indicates what account-type it is?

Reply:
Good idea